### PR TITLE
Update glpk recipe from anaconda channel

### PR DESCRIPTION
**Pull request by vivekiitkgp**
Signed-off-by: Vivek Rai <vivekraiiitkgp@gmail.com>

### CODE REVIEW

**Comment by vivekiitkgp** [comment]
Any idea why it might be failing? As I see, gmp recipe is already there in anaconda channel.

**Comment by johanneskoester** [comment]
Yes, you need to make sure that the compiler sees the includes from conda. You should strictly use the gcc from conda like in [this](https://github.com/bioconda/bioconda-recipes/tree/32a9dabfe1a6cc4e2a14f9f25e26219d805a26df/recipes/clark) recipe. Note that you first depend on gcc and libgcc in the meta.yaml. Second, you define the environment variables as in the build.sh.

**Comment by vivekiitkgp** [comment]
@johanneskoester Looks okay to me. Review?

File: recipes/glpk/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
+
   number: 0
 
+requirements:
+  build:
+    - gmp      [not win]
```
Comment (johanneskoester) [inline recipes/glpk/meta.yaml]: We also need gcc as a build dependency and libgcc as a run dependency here. We don't build for win, so you could elimininate the windows stuff for simplicity. But that's up to you. Further, according to the [docs](http://conda.pydata.org/docs/building/meta-yaml.html#skipping-builds), it is best practice to have a ``#`` in front of the selectors.

**Comment by vivekiitkgp** [comment]
@johanneskoester Updated. I kept the Windows config.

**Comment by johanneskoester** [comment]
Ok, thanks!

**Pull request merged**